pkg/config/system_probe.go

```diff
@@ -117,6 +117,7 @@ func InitSystemProbeConfig(cfg Config) {
 	cfg.BindEnvAndSetDefault(join(spNS, "closed_channel_size"), 500)
 	cfg.BindEnvAndSetDefault(join(spNS, "max_connection_state_buffered"), 75000)
 
+	cfg.BindEnvAndSetDefault(join(spNS, "disable_protocol_classification"), false, "DD_DISABLE_PROTOCOL_CLASSIFICATION")
```

Comment:
Do we want this to be affirmative ?
When I did the windows side, I did the same change but was
`enable_protocol_classification`.
I think the "polarity" doesn't matter as much as the default.

Reply:
In general, enable-style flags are preferred, because they are easier to understand. Doing `disable=false` to enable, is a bit confusing.

Reply:
i'll make the change, with enabled = true by default

I was following the "disable_dns" flag notation